sql/system.go

```diff
@@ -61,6 +61,19 @@ CREATE TABLE system.zones (
   id     INT PRIMARY KEY,
   config BYTES
 );`
+
+	// Cluster Reporting table schema
+	reportingTableSchema = `
+CREATE TABLE system.reporting (
+  email       STRING PRIMARY KEY,
+  optin       BOOLEAN,
+  dismissed   INT,
+  firstname   STRING,
+  lastname    STRING,
+  company     STRING,
+  updates     BOOLEAN,
+  lastUpdated TIMESTAMP
+);`
```

Comment:
This makes sense and seems better than what I'm doing. I'd maybe just add a user column for when we have per-user settings (it would be null for reporting.)